docs/markdown/foundations/content_standards/ui_elements.md

```diff
@@ -0,0 +1,194 @@
+---
+title: UI Elements
+description: Here are a few guidelines we recommend you follow when writing near UI elements to help make things clear
+fullwidth: true
+---
+
+## Buttons
+Buttons should be descriptive but short.
+
+<TwoCol>
+  <Group>
+    <Do title="Do"/>
+     - If your object is already described on the screen, buttons only need a verb
+     - If your object isn’t described on the screen, buttons need a verb + the object
+     - Use fewer than 3 words
+     - Use sentence case
+     - Examples:
+        - Save
+        - Edit profile
+        - See job openings
+        - Cancel
+        - No thanks 
+  </Group>
+  <Group>
+  <Dont title="Don't"/>
+     - Don’t use punctuation
+     - Examples:
+        - Save changes
+        - Edit
+        - Jobs
+        - No, thanks
+  </Group>
+</TwoCol>
+
+---
```

Comment:
Sorry comment was in the wrong spot, there are these line breaks that need to be removed